PencilCase/PencilCase/scripts/Models/Order.ts

```diff
@@ -49,6 +54,7 @@ export class Order {
 
 export enum OrderTypes {
     Retail = 1,
-    Wholesale,
-    Import
+    Wholesale, //2
+    Import, //3
+    RetailWholesale //4
```

Comment:
The actual number value will be used when query data from database in Order Repository.